eventsdiscovery/src/main/java/com/schedjoules/client/eventsdiscovery/http/SingleEventResponseHandler.java

```diff
@@ -56,21 +58,14 @@ public String etag()
             @Override
             public String uid()
             {
-                return payload().uid();
+                return payload().value().uid();
```

Comment:
Could this class reuse the code from `MultiEventsResponseHander` to create the `Envelope`?